autorest/adal/msi_windows.go

```diff
@@ -0,0 +1,6 @@
+// +build windows
+
+package adal
+
+// msiPath is the path to the MSI Extension settings file (to discover the endpoint)
+var msiPath = "C:/WindowsAzure/Config/ManagedIdentity-Settings"
```

Comment:
We still shouldn't assume that this file lives on the `C` drive :) 

You could dynamically generate this like the following: https://play.golang.org/p/RtnOwH5tGu

Reply:
Fixed